src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -134,6 +136,126 @@ private VariantOverlapAnnotator initializeOverlapAnnotator(final FeatureInput<Va
         return descriptions;
     }
 
+    //TODO the following methods are not hooked up to any tools
+    /**
+     * Combine (raw) data for reducible annotations (those that use raw data in gVCFs)
+     * Mutates annotationMap by removing the annotations that were combined
+     * @param allelesList   the list of merged alleles across all variants being combined
+     * @param annotationMap attributes of merged variant contexts -- is modifying by removing successfully combined annotations
+     * @return  a map containing the keys and raw values for the combined annotations
+     */
+    //@SuppressWarnings()
+    public Map<String, Object> combineAnnotations(final List<Allele> allelesList, Map<String, List<ReducibleAnnotationData<Object>>> annotationMap) {
+        Map<String, Object> combinedAnnotations = new HashMap<>();
+
+        // go through all the requested reducible info annotationTypes
+        for (final InfoFieldAnnotation annotationType : infoAnnotations) {
+            if (annotationType instanceof ReducibleAnnotation) {
+                ReducibleAnnotation currentASannotation = (ReducibleAnnotation) annotationType;
+                if (annotationMap.containsKey(currentASannotation.getRawKeyName())) {
+                    final List<ReducibleAnnotationData<Object>> annotationValue = annotationMap.get(currentASannotation.getRawKeyName());
```

Comment:
It's usually better to use wildcards, eg., `ReducibleAnnotationData<?>`, rather than `Object` as a type parameter.